torch/sparse/triton_ops/triton_bsr_dense_mm.py

```diff
@@ -0,0 +1,295 @@
+import torch
+import triton
+import triton.language as tl
+
+
+def compressed_indices_to_plain_indices(cidx, pidx):
```

Comment:
nit: is this unused?

Reply:
Not here anymore, right, but it is a very nice util for other kernels we might wanna have for sparse in Triton. We can remove it now...